bundles/framework/layerlist/Tile.js

```diff
@@ -0,0 +1,140 @@
+     * @static
+     * @param {Oskari.mapframework.bundle.layerlist.LayerSelectionBundleInstance}
+     * instance
+     *      reference to component that created the tile
+     */
+    function (instance) {
+        this.instance = instance;
+        this.container = null;
+        this.template = null;
+        this.shownLayerCount = null;
+    }, {
+    /**
+     * @method getName
+     * @return {String} the name for the component
+     */
+        getName: function () {
+            return 'Oskari.mapframework.bundle.layerlist.Tile';
+        },
+        /**
+         * @method setEl
+         * @param {Object} el
+         *      reference to the container in browser
+         * Interface method implementation
+         */
+        setEl: function (el) {
+            this.container = jQuery(el);
+        },
+        /**
+         * @method startPlugin
+         * Interface method implementation, calls #refresh()
+         */
+        startPlugin: function () {
+            this._addTileStyleClasses();
+            this.refresh();
+        },
+        _addTileStyleClasses: function () {
+            const isContainer = !!((this.container && this.instance.mediator));
+            const isBundleId = !!((isContainer && this.instance.mediator.bundleId));
+            const isInstanceId = !!((isContainer && this.instance.mediator.instanceId));
+
+            if (isInstanceId && !this.container.hasClass(this.instance.mediator.instanceId)) {
+                this.container.addClass(this.instance.mediator.instanceId);
+            }
+            if (isBundleId && !this.container.hasClass(this.instance.mediator.bundleId)) {
+                this.container.addClass(this.instance.mediator.bundleId);
+            }
+        },
+        /**
+         * @method stopPlugin
+         * Interface method implementation, clears the container
+         */
+        stopPlugin: function () {
+            this.container.empty();
+        },
+        /**
+         * @method getTitle
+         * @return {String} localized text for the title of the tile
+         */
+        getTitle: function () {
+            return this.instance.getLocalization('title');
+        },
+        /**
+         * @method getDescription
+         * @return {String} localized text for the description of the tile
+         */
+        getDescription: function () {
+            return this.instance.getLocalization('desc');
+        },
+        /**
+         * @method setState
+         * @param {Object} state
+         *      state that this component should use
+         * Interface method implementation, does nothing atm
+         */
+        setState: function (state) {
+        },
+        notifyUser: function () {
+            const status = this.container.children('.oskari-tile-status');
+
+            // stop current animation
+            status.stop();
+            // blink 2 times
+            this._blink(status, 2);
+        },
+        _blink: function (element, count) {
```

Comment:
Try to implement this with CSS rather than jQuery. CSS has been used in thematic maps for instance: https://github.com/oskariorg/oskari-frontend/blob/1.53.1/bundles/statistics/statsgrid2016/resources/scss/style.scss#L338-L343